docs/source/index.rst

```diff
@@ -45,6 +45,68 @@ This section details the list of released packages for each Daml SDK release. It
+| Daml.Finance.Holding                       | 1.0.2              | Stable |
++--------------------------------------------+--------------------+--------+
+| Daml.Finance.Instrument.Generic            | 1.0.1              | Stable |
++--------------------------------------------+--------------------+--------+
+| Daml.Finance.Instrument.Token              | 1.0.1              | Stable |
++--------------------------------------------+--------------------+--------+
+| Daml.Finance.Interface.Data                | 1.0.1              | Stable |
++--------------------------------------------+--------------------+--------+
+| Daml.Finance.Interface.Types.Date          | 2.0.0              | Stable |
++--------------------------------------------+--------------------+--------+
+| Daml.Finance.Lifecycle                     | 1.0.1              | Stable |
++--------------------------------------------+--------------------+--------+
+| Daml.Finance.Settlement                    | 1.0.2              | Stable |
++--------------------------------------------+--------------------+--------+
+| Daml.Finance.Util                          | 2.0.0              | Stable |
++--------------------------------------------+--------------------+--------+
+
+Early Access Packages
+---------------------
+
++--------------------------------------------+--------------------+--------+
+| Package                                    | Version            | Status |
++============================================+====================+========+
+| Daml.Finance.Instrument.Bond               | 0.2.1              | Alpha  |
++--------------------------------------------+--------------------+--------+
+| Daml.Finance.Instrument.Equity             | 0.2.1              | Alpha  |
++--------------------------------------------+--------------------+--------+
+| Daml.Finance.Instrument.Option             | 0.1.0              | Alpha  |
++--------------------------------------------+--------------------+--------+
+| Daml.Finance.Instrument.Swap               | 0.2.1              | Alpha  |
++--------------------------------------------+--------------------+--------+
+| Daml.Finance.Interface.Instrument.Bond     | 0.2.1              | Alpha  |
++--------------------------------------------+--------------------+--------+
+| Daml.Finance.Interface.Instrument.Option   | 0.1.0              | Alpha  |
++--------------------------------------------+--------------------+--------+
+| Daml.Finance.Interface.Instrument.Swap     | 0.2.1              | Alpha  |
++--------------------------------------------+--------------------+--------+
+
+Deprecated Packages
+-------------------
+
++--------------------------------------------+--------------------+--------+
+| Package                                    | Version            | Status |
++============================================+====================+========+
+| None                                                                     |
```

Comment:
Are we deprecating the `1.0` versions of `D.F.Util` and `D.F.I.Types.Date`?

Reply:
I was thinking about that as well. I suggest that we do. I wrote version `1.*` as a general expression to use in the future as well, since there may be several 1.* versions.